plone/app/jquerytools/browser/jquery.form.js

```diff
@@ -10,6 +10,21 @@
  */
 /*global ActiveXObject */
 
+// AMD support
```

Comment:
Are you sure, you included jquery.form 3.51.0 ? It looks like, the only change here is AMD support.